Test/Service.cs

```diff
@@ -96,6 +98,32 @@ public static class Util {
                 {"description", description}
             };
         }
+        
+        public static Dictionary<string, object> PreprocessConfig(){
+            return new Dictionary<string, object>{
+                {"autoRotate", true},
+                {"maxPages", 3},
+                {"imageQuality", "HIGH"}
+            };
+        }
+            
+        public static Dictionary<string, object> FieldConfig(){
+            return new Dictionary<string, object>{
+                {"DueDate", new Dictionary<string, object>(){
+                    {"type", "date"},
+                    {"description", "due date of the invoice"},
+                    {"maxLength", 11}
+                    }
+                },
+                {"TotalAmount", new Dictionary<string, object>(){
```

Comment:
(Foreach Dictionary in code) Do we not want a nice litte whitespace between > and {? Like so:

```suggestion
                {"TotalAmount", new Dictionary<string, object> {
```